site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+   - **Description**: Provide details about the project.
+     This field is visible only when the **Use project description** toggle is off.
+   - **Sensible content**: Switch this toggle if your dataset
+     contains images intended for an adult audience.
+   - **Accept solutions automatically**: Enable this if you
+     wish for completed jobs to be automatically accepted.
+   - **Close pool after completion, sec**: Enter the time limit, in seconds,
+     within which each job should be completed. You can also click on the
+     field to select from one of the predefined periods.
+   - **Time per task suite, sec**: Enter the time allowed for completing a task suite,
+     in seconds. Uncompleted tasks are redistributed to other Tolokers.
+   - **Days for manual review by requester**: Specify the Review period in days —
+     the number of days for the review (from 1 to 21 days from the task completion date).
+     The Toloker will see the deadline in the task information on the main Toloka page.
+   - **Audience**: Add rules to make jobs available only to Tolokers who meet certain criteria.
+     For example, you might require Tolokers to be proficient in English and have higher education.
+     These rules operate based on filter principles. For more information,
+     see [Filters](http://localhost:1313/docs/manual/advanced/filter/) documentation
+     or [CVAT Filters](https://www.youtube.com/watch?v=lj6KLIFn24A) video tutorial.
+   - ![Toloka Rules](/images/toloka10.jpg)
+
+3. Click **Submit**. You will see the **Toloka task was created** pop-up and the **Update pool** form.
+
+   ![Update Project](/images/toloka11.jpg)
+
+   - ![Open pool in Toloka](/images/toloka04.png) opens pool in Toloka.
+   - ![Open task in CVAT](/images/toloka05.png) opens task in CVAT.
+
+4. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+   <br>Project, that you created will show up in Toloka.</br>
+
+   ![Toloka Project](/images/toloka12.jpg)
+
+## Changing Toloka pool
+
+To change started Toloka pool, you need to stop it first.
+
+1. Open Toloka task, **Actions** > **Stop Toloka pool**.
+2. Implement changes.
+3. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+
+## Reviewing annotated jobs
+
+In case the jobs you've created are not in the **Accept solutions automatically**
+mode, you will need to manually review and accept them
+within time limits that were defined in the task setting.
+
+To approve or reject the job, use the **Accept** and **Reject** buttons.
+
+![Toloka Project](/images/toloka13.jpg)
+
+### Accepting job
+
+To accept the annotated job, do the following:
+
+1. Go to the Toloka task and open the job.
+2. Review the result of annotation and in case all is fine, on the top menu,
+   click **Accept**.
+3. Add comment.
+
+   ![Toloka Project](/images/toloka14.jpg)
+
+4. Click **OK**.
+
+### Rejecting job
+
+To accept the annotated job, do the following:
```

Comment:
We need to add note, that rejected job can be appelated by Toloker on Toloka platform